### PR TITLE
Update dual-shipping.md

### PR DESCRIPTION
**Pull request by mrdoggopat**
Add missing configurations in the docs to fully dual ship cluster agent metrics and Orchestrator Explorer Data

### What does this PR do? What is the motivation?

Add missing configurations in the docs to fully dual ship cluster agent metrics and Orchestrator Explorer Data

### Merge instructions

- [✅] Please merge after reviewing

### CODE REVIEW

File: content/en/agent/guide/dual-shipping.md [hunk 1]
```diff
@@ -127,11 +127,33 @@ Requires Agent version >= 6.20 or 7.20.
 DD_PROCESS_ADDITIONAL_ENDPOINTS='{\"https://process.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://process.datadoghq.eu\": [\"apikey4\"]}'
 ```
 
+## Cluster Agent metrics (e.g. Kubernetes State Metrics Core)
+
```
Comment (maycmlee) [inline content/en/agent/guide/dual-shipping.md:131]:
```suggestion
Configure the Agent to send Cluster Agent metrics, such as Kubernetes State Metrics Core, to additional endpoints.
```

File: content/en/agent/guide/dual-shipping.md [hunk 2]
```diff
@@ -127,11 +127,33 @@ Requires Agent version >= 6.20 or 7.20.
 DD_PROCESS_ADDITIONAL_ENDPOINTS='{\"https://process.datadoghq.com\": [\"apikey2\", \"apikey3\"], \"https://process.datadoghq.eu\": [\"apikey4\"]}'
 ```
 
+## Cluster Agent metrics (e.g. Kubernetes State Metrics Core)
```
Comment (maycmlee) [inline content/en/agent/guide/dual-shipping.md:130]:
```suggestion
## Cluster Agent metrics
```

**Review by maycmlee — CHANGES_REQUESTED** [review]
Made a couple of suggestions. Let me know if it doesn't work.

File: content/en/agent/guide/dual-shipping.md (hunk 1 above)
Comment (mrdoggopat) [reply content/en/agent/guide/dual-shipping.md:131]: IGTM

File: content/en/agent/guide/dual-shipping.md (hunk 2 above)
Comment (mrdoggopat) [reply content/en/agent/guide/dual-shipping.md:130]: IGTM

Comment (maycmlee) [inline content/en/agent/guide/dual-shipping.md:130]:
```suggestion
## Cluster Agent metrics
```

File: content/en/agent/guide/dual-shipping.md (hunk 1 above)
Comment (maycmlee) [inline content/en/agent/guide/dual-shipping.md:131]:
```suggestion
Configure the Agent to send Cluster Agent metrics, such as Kubernetes State Metrics Core, to additional endpoints.
```

**Review by maycmlee — APPROVED** [review]

**Pull request merged**